src/node_api.cc

```diff
@@ -123,6 +187,22 @@ v8::Local<v8::Value> V8LocalValueFromJsValue(napi_value v) {
   return local;
 }
 
+napi_status V8NameFromPropertyDescriptor(napi_env env,
```

Comment:
How about either adding `static inline` here too (and for helpers in general), or – and I think I would prefer that – turning `v8impl` into an anonymous namespace?